README.md

```diff
@@ -41,6 +42,7 @@ disk_additional_disks:
 * `mount` is the directory where the new disk should be mounted.
 * `user` sets owner of the mount directory (default: `root`).
 * `group` sets group of the mount directory (default: `root`).
+* `mode` sets the mountpoint directory permissions (default: `0775`).
```

Comment:
Could we make the default `755`?

Reply:
Of course, my bad for that permissive default 😸